lib/MiLight/MiLightClient.h

```diff
@@ -14,38 +16,53 @@
 
 class MiLightClient {
   public:
-    MiLightClient(uint8_t cePin, uint8_t csnPin)
-      : rf(RF24(cePin, csnPin)),
-      resendCount(MILIGHT_DEFAULT_RESEND_COUNT),
-      currentRadio(NULL),
-      numRadios(MiLightRadioConfig::NUM_CONFIGS)
+  MiLightClient(byte byCSPin, byte byResetPin, byte byPktFlag, eRadioInterfaceType InterfaceType)
+  :
+  resendCount(MILIGHT_DEFAULT_RESEND_COUNT),
+  currentRadio(NULL),
+  numRadios(MiLightRadioConfig::NUM_CONFIGS)
+
+  {
+    radios = new RadioStack*[numRadios];
+
+    for (size_t i = 0; i < numRadios; i++)
     {
-      radios = new RadioStack*[numRadios];
-      
-      for (size_t i = 0; i < numRadios; i++) {
-        radios[i] = new RadioStack(rf, *MiLightRadioConfig::ALL_CONFIGS[i]);
+      if(InterfaceType == PL1167_LT8900)
```

Comment:
Good idea, - I went after prove of concept - feel free mock around with the code.